addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```
@@ -74,19 +64,17 @@ $o-hr-holidays-border-color: map-get($grays, '300');
                     margin-left: 22px;
                     margin-right: 2px;
                     width: 500px;
-                    border-radius: 3px;
-                    background-color: $o-gray-200;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
 
                     // Triangle
                     &:before {
                         content: '';
-                        @include o-position-absolute($top: 12px, $left: -17px);
+                        @include o-position-absolute($top: 12px, $left: -18px);
                         margin-left: 10px;
                         width: 14px;
                         height: 14px;
-                        background-color: $o-gray-200;
-                        border-bottom: 1px solid $o-hr-holidays-border-color;
+                        background: $o-view-background-color;
+                        border-bottom: 1px solid $gray-300;
```

To maintains consistency with borders you can use border variables :) 
```suggestion
                        border-bottom: $border-width solid $border-color;
```
Same for the line below 👇